testsuite/gnat2goto/tests/loops/loops.adb

```diff
@@ -44,4 +44,9 @@ begin
    while z > 1 loop
       exit;
    end loop;
+
+   for i in 1..10 loop
+     pragma assert(i < 10);
+     null;
+   end loop;
```

Comment:
This part was meant to test loop unwinding, I don't think it's really necessary anymore.